src/App.jsx

```diff
@@ -11,10 +11,12 @@ import { SummaryPage } from "./components/SummaryPage";
 import { PrepareSubmitPage } from "./components/PrepareSubmitPage";
 import { AllForms } from "./components/AllForms";
 import { FormPageWrapper } from "./components/FormPageWrapper";
-import navdesign from "template";
+import "./overrideFormioStyles.less";
+
+const components = CustomComponents;
```

Comment:
Unneccessary to rename CustomComponents directly before passing it as an argument to Components.setComponents